### ISSUE TITLE
Line length limit

### ISSUE THREAD

**Issue by mvz** [troessner/reek#347]
This is a follow-up to https://github.com/troessner/reek/pull/328#discussion_r21415725, where @troessner wrote:

> I wrapped it to 80 characters now. Did we ever set on a standard here?
> Personally I find 80 a little limiting and would rather go for 120.

Regarding line length, there are several factors I think. For a block of text to be read like the above, 80 characters is a nice limit because it's just a lot easier to read.

For code, 80 characters is indeed a bit too limiting as a hard limit. I generally let lines go up to 90 or 100 characters before wrapping, but then wrap to about 80.

I also find anything that makes me have to scroll diffs horizontally on GitHub annoying, so that sets some limit of around 100 or so.

Our RuboCop config is set at a 92 character limit, by the way.

In conclusion, my proposal would be to have a 'hard' RuboCop limit of somewhere between 92 and 100 characters, and wrap to around 80 if wrapping is needed.
